components/event-feed-service/pkg/integration_test/feed_test.go

```diff
@@ -222,128 +222,6 @@ func TestEventFeedReturnOnlyEventsWithinDateRange(t *testing.T) {
 	}
 }
 
-func TestEventFeedFilterTags(t *testing.T) {
```

Comment:
The tags field is not going to be used for filtering, so there is currently no way to add `org_name` to an event.